magefiles/test.go

```diff
@@ -12,12 +12,18 @@ type Test mg.Namespace
 
 // All runs all test suites
 func (t Test) All() error {
-	mg.Deps(t.Integration)
+	mg.Deps(t.Integration, t.Unit)
 	return nil
 }
 
-// Integration runs the unit tests
-func (Test) Integration() error {
+// Unit runs the unit tests
+func (t Test) Unit() error {
 	fmt.Println("running unit tests")
-	return goTest("./...", "-tags", "integration", "-timeout", "10m")
+	return goTest("./...", "-race", "-count", "1", "-timeout", "20m")
+}
+
+// Integration runs the integration tests
+func (Test) Integration() error {
```

Comment:
The unit tests have a timeout of 20m, but the integration tests have a timeout of 1m? No race detector?